ci-operator/config/openshift/installer/openshift-installer-main.yaml

```diff
@@ -489,6 +489,42 @@ tests:
       AWS_METADATA_SERVICE_AUTH: Required
     workflow: openshift-e2e-aws
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-byo-subnet-role
```

Comment:
How about combine custom sg to catch https://issues.redhat.com/browse/OCPBUGS-55324

Reply:
Updated.